presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -175,8 +175,7 @@ void PrestoServer::run() {
     }
     nodeLocation_ = nodeConfig->nodeLocation();
   } catch (const VeloxUserError& e) {
-    // VeloxUserError is always logged as an error.
-    // Avoid logging again.
+    LOG(ERROR) << e.what();
```

Comment:
Can you give more context here on error?
```
LOG(ERROR) << "...: " << e.what();
```

Reply:
May be "Failed to start server due to " << e.what()

Reply:
Velox exception prints a good amount of detail, hence I did not add any prefix. Below is the printout for this case:

```
I20230831 23:16:48.217065 328279 Configs.cpp:210] [PRESTO_STARTUP] Registered properties from '/Users/adutta/Documents/code/presto/presto-native-execution/etc/node.properties':
  node.internal-address=127.0.0.1
  node.location=testing-location
  node.id=e4901aae-a1c9-4ff7-97a9-5687835ad54c
  node.environment=testing
E20230831 23:16:48.217298 328279 PrestoServer.cpp:178] Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: Couldn't open config file ~/presto/presto-native-execution/etc/velox.properties for reading.
Retriable: False
Function: readConfig
File:~/presto/presto-native-execution/presto_cpp/main/common/ConfigReader.cpp
Line: 28
Stack trace:
Stack trace has been disabled.Use --velox_exception_user_stacktrace=true to enable it.
```